adapter/src/main/java/ar/edu/undec/adapter/data/entity/CourseDTO.java

```diff
@@ -0,0 +1,41 @@
+package ar.edu.undec.adapter.data.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import curso.modelo.Level;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CourseDTO {
+    @JsonProperty("id")
+    private UUID id;
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("Inscription_deadLine")
```

Comment:
Solo por convención snake_lower_case

```suggestion
    @JsonProperty("inscription_deadline")
```